Tasks/XamariniOS/xcode-task-utils.js

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
This utils file is copied from XCode task as is since there isn't a way to share them today